### DISCUSSION TITLE
EXE file won't run if the folder path contain non ascii character

### DISCUSSION THREAD

**Discussion by maj01** [pyinstaller/pyinstaller#5507] [PyInstaller] [answered]
Hello

If i put the produced .exe in a path that contains Arabic letters it won't run. and it will produce the error:
[https://imgur.com/uBbXBMi
](url)

can this be fixed?

**Comment by bwoodsend** [comment, marked as answer]
It looks like tkinter isn't unicode-safe. In which case no it can't be fixed.

**Reply by maj01** [reply]
Thank You!
